src/Symfony/Component/Routing/Loader/Configurator/Traits/RouteTrait.php

```diff
@@ -95,7 +95,7 @@
      * Sets the schemes (e.g. 'https') this route is restricted to.
      * So an empty array means that any scheme is allowed.
      *
-     * @param array $schemes
+     * @param string|string[] $schemes
```

Comment:
doesn't match the type hint of the function, isn't it? should it be changed? (same below)

Reply:
The `Route::setSchemes()` methods docblock says `string|array`. Which could indeed be fixed in lower branches to `string|string[]`. Same for `Route::setMethods()`.

Reply:
See https://github.com/symfony/symfony/pull/24351

Reply:
The type hint in the current still doesn't allow strings.

Reply:
Oh! My bad >< Sorry about the confusion. No, let's keep it as is. Saving two chars for declaring a single string instead of an array isn't worth the name mismatch to me. Juste changed to `string[]`